src/main/java/mate/jdbc/service/CarServiceImpl.java

```diff
@@ -0,0 +1,63 @@
+package mate.jdbc.service;
+
+import java.util.List;
+import mate.jdbc.dao.CarDao;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+
+@Service
+public class CarServiceImpl implements CarService {
+    @Inject
+    private CarDao carDao;
+
+    @Override
+    public Car create(Car car) {
+        return carDao.create(car);
+    }
+
+    @Override
+    public Car get(Long id) {
+        return carDao.get(id).orElseThrow(() ->
+                new DataProcessingException("No car with such id = " + id));
+    }
+
+    @Override
+    public List<Car> getAll() {
+        return carDao.getAll();
+    }
+
+    @Override
+    public Car update(Car car) {
+        return carDao.update(car);
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        return carDao.delete(id);
+    }
+
+    @Override
+    public void addDriverToCar(Driver driver, Car car) {
+        car.getDrivers().add(driver);
+        carDao.update(car);
+    }
+
+    @Override
+    public void removeDriverFromCar(Driver driver, Car car) {
+        if (car.getDrivers() != null) {
+            car.getDrivers().remove(driver);
+            carDao.update(car);
+        } else {
+            throw new DataProcessingException("This car " + car
+                    + " doesn't have such driver " + driver);
+        }
```

Comment:
just a better practice of `if`s
```suggestion
        if (car.getDrivers() == null) {
            throw new DataProcessingException("This car " + car
                    + " has null drivers list");
        }
        car.getDrivers().remove(driver);
        carDao.update(car);
```